lib/cli/bin/doc/licenseList.js

```diff
@@ -8,6 +8,8 @@ var spdxCodes = require('spdx-license-list');
 var ejs = require('ejs');
 var program = require('commander');
 
```

Comment:
can we just remvoe it?

Reply:
yes, we can remove, was not sure which scripts are using it

Reply:
ok, removed old command

Reply:
we can then check it